diff --git a/node_modules/@contena/meteor-component-library/dist/esm/MtDataTable.js b/node_modules/@contena/meteor-component-library/dist/esm/MtDataTable.js
index c0ff6c0..d40e41e 100644
--- a/node_modules/@contena/meteor-component-library/dist/esm/MtDataTable.js
+++ b/node_modules/@contena/meteor-component-library/dist/esm/MtDataTable.js
@@ -1418,9 +1418,9 @@
               ]) : createCommentVNode("", true),
               _ctx.allowRowSelection ? withDirectives((openBlock(), createElementBlock("th", _hoisted_10, [
                 createVNode(_component_mt_checkbox, {
-                  checked: _ctx.somethingSelected,
+                  modelValue: _ctx.somethingSelected,
                   onChange: _ctx.handleSelectAll
-                }, null, 8, ["checked", "onChange"])
+                }, null, 8, ["modelValue", "onChange"])
               ])), [
                 [_directive_stickyColumn]
               ]) : createCommentVNode("", true),
@@ -1620,10 +1620,10 @@
                 ]) : createCommentVNode("", true),
                 _ctx.allowRowSelection ? withDirectives((openBlock(), createElementBlock("td", _hoisted_23, [
                   createVNode(_component_mt_checkbox, {
-                    checked: _ctx.getSelectionValue(data.id),
+                    modelValue: _ctx.getSelectionValue(data.id),
                     disabled: _ctx.disableRowSelect.includes(data.id),
                     onChange: ($event) => _ctx.onRowSelect(data.id)
-                  }, null, 8, ["checked", "disabled", "onChange"])
+                  }, null, 8, ["modelValue", "disabled", "onChange"])
                 ])), [
                   [_directive_stickyColumn]
                 ]) : createCommentVNode("", true),
diff --git a/node_modules/@contena/meteor-component-library/dist/common/MtDataTable.js b/node_modules/@contena/meteor-component-library/dist/common/MtDataTable.js
index 084353d..2ecdaab 100644
--- a/node_modules/@contena/meteor-component-library/dist/common/MtDataTable.js
+++ b/node_modules/@contena/meteor-component-library/dist/common/MtDataTable.js
@@ -1419,9 +1419,9 @@
               ]) : vue.createCommentVNode("", true),
               _ctx.allowRowSelection ? vue.withDirectives((vue.openBlock(), vue.createElementBlock("th", _hoisted_10, [
                 vue.createVNode(_component_mt_checkbox, {
-                  checked: _ctx.somethingSelected,
+                  modelValue: _ctx.somethingSelected,
                   onChange: _ctx.handleSelectAll
-                }, null, 8, ["checked", "onChange"])
+                }, null, 8, ["modelValue", "onChange"])
               ])), [
                 [_directive_stickyColumn]
               ]) : vue.createCommentVNode("", true),
@@ -1621,10 +1621,10 @@
                 ]) : vue.createCommentVNode("", true),
                 _ctx.allowRowSelection ? vue.withDirectives((vue.openBlock(), vue.createElementBlock("td", _hoisted_23, [
                   vue.createVNode(_component_mt_checkbox, {
-                    checked: _ctx.getSelectionValue(data.id),
+                    modelValue: _ctx.getSelectionValue(data.id),
                     disabled: _ctx.disableRowSelect.includes(data.id),
                     onChange: ($event) => _ctx.onRowSelect(data.id)
-                  }, null, 8, ["checked", "disabled", "onChange"])
+                  }, null, 8, ["modelValue", "disabled", "onChange"])
                 ])), [
                   [_directive_stickyColumn]
                 ]) : vue.createCommentVNode("", true),
diff --git a/node_modules/@contena/meteor-component-library/dist/esm/MtPopoverItem.js b/node_modules/@contena/meteor-component-library/dist/esm/MtPopoverItem.js
index f672623..008d09d 100644
--- a/node_modules/@contena/meteor-component-library/dist/esm/MtPopoverItem.js
+++ b/node_modules/@contena/meteor-component-library/dist/esm/MtPopoverItem.js
@@ -244,10 +244,10 @@
         key: 0,
         id: _ctx.id,
         class: "mt-popover-item__checkbox",
-        checked: _ctx.checkboxChecked,
+        modelValue: _ctx.checkboxChecked,
         partial: _ctx.checkboxPartial,
         onChange: _ctx.handleLabelClick
-      }, null, 8, ["id", "checked", "partial", "onChange"])) : createCommentVNode("", true),
+      }, null, 8, ["id", "modelValue", "partial", "onChange"])) : createCommentVNode("", true),
       renderSlot(_ctx.$slots, "extension-logo"),
       _ctx.icon ? (openBlock(), createBlock(_component_mt_icon, {
         key: 1,
diff --git a/node_modules/@contena/meteor-component-library/dist/common/MtPopoverItem.js b/node_modules/@contena/meteor-component-library/dist/common/MtPopoverItem.js
index ffc0aef..49e240c 100644
--- a/node_modules/@contena/meteor-component-library/dist/common/MtPopoverItem.js
+++ b/node_modules/@contena/meteor-component-library/dist/common/MtPopoverItem.js
@@ -245,10 +245,10 @@
         key: 0,
         id: _ctx.id,
         class: "mt-popover-item__checkbox",
-        checked: _ctx.checkboxChecked,
+        modelValue: _ctx.checkboxChecked,
         partial: _ctx.checkboxPartial,
         onChange: _ctx.handleLabelClick
-      }, null, 8, ["id", "checked", "partial", "onChange"])) : vue.createCommentVNode("", true),
+      }, null, 8, ["id", "modelValue", "partial", "onChange"])) : vue.createCommentVNode("", true),
       vue.renderSlot(_ctx.$slots, "extension-logo"),
       _ctx.icon ? (vue.openBlock(), vue.createBlock(_component_mt_icon, {
         key: 1,
